libs/components/src/next/button/vl-button.css.ts

```diff
@@ -0,0 +1,234 @@
+import { mediaQuerySmall } from '@domg-wc/common-utilities/css/mixin/media-queries.mixin.css';
+import { focusOutline } from '@domg-wc/common-utilities/css/mixin/outline.mixin.css';
+import { waveAnimation } from '@domg-wc/common-utilities/css/mixin/animation.mixin.css';
+import { CSSResult, css, unsafeCSS } from 'lit';
+
+const borderWidth = '0.2rem';
+const borderWidthSmall = '0.1rem';
+const differenceBetweenRegularBorderAndSmallBorder = `calc(${borderWidth} - ${borderWidthSmall})`;
+const loadingAnimationName = 'waving-light';
+
+const styles: CSSResult = css`
+    /* Importeer loading animation, moet op dit niveau geïmporteerd worden. */
+    ${waveAnimation(loadingAnimationName, 'var(--vl-action-color--disabled)')}
+
+    button {
+        /* Reset styles - gebaseerd op DV mixin > _buttons.scss */
+        border-radius: 0;
+        appearance: none;
+        -webkit-appearance: none; // doesn't work without prefix on Safari iOS11
+        border: 0;
+        background-color: transparent;
+        padding: 0;
+
+        /* Button styles - gebaseerd op DV _button.scss */
+        cursor: pointer;
+        display: inline-flex;
+        align-items: center;
+        min-height: 3.5rem;
+        font-family: inherit;
+        font-size: var(--vl-font-size--small);
+        font-weight: 500;
+        padding: var(--vl-spacing--xxsmall) var(--vl-spacing--normal);
+        background-color: var(--vl-action-color);
+        border: 0;
+        border-radius: var(--vl-border-radius);
+        color: var(--vl-white);
+        max-width: 100%;
+
+        ${mediaQuerySmall(
+            css`
+                padding: var(--vl-spacing--xsmall);
+            `
+        )}
+
+        &:focus {
+            ${focusOutline()}
+        }
+
+        &:hover {
+            background-color: var(--vl-action-color--hover);
+        }
+
+        &:active {
+            background-color: var(--vl-action-color--active);
+        }
+
+        &.secondary {
+            color: var(--vl-action-color);
+            background-color: transparent;
+            border: ${unsafeCSS(borderWidth)} solid currentColor;
+            transition: color 0.2s, border-color 0.2s;
+
+            &:focus {
+                color: var(--vl-action-color--active);
+            }
+
+            &:hover,
+            &:active {
+                color: var(--vl-action-color--hover);
+            }
+        }
+
+        &.tertiary {
+            color: var(--vl-action-color);
+            background-color: transparent;
+            border: ${unsafeCSS(borderWidthSmall)} var(--vl-action-tertiary-border-color) solid;
+            transition: background-color 0.2s;
+
+            &:focus {
+                color: var(--vl-action-tertiary-color--hover);
+                border-color: var(--vl-action-tertiary-border-color);
+            }
+
+            &:hover,
+            &:active {
```

Comment:
Ondanks eerdere discussies waarbij we afgesproken hebben (tegen jou mening in) om borderless styling van primary styling te behouden, vind ik het nu ik het zie in storybook, toch een beetje storend dat bij toggelen en hoveren over de button het verspringt en neig ik nu toch eerder naar jouw voorstel om ervoor te zorgen dat dit visueel niet vespringt.

Aan jou om dit eventueel nog mee te nemen of niet.
@kspeltix Wat denk jij?

![toggle-verspringen](https://github.com/milieuinfo/uigov-web-components/assets/15731348/1b3228d3-a4cc-43e0-9485-52997ed6470a)